src/skin/firstRun.html

```diff
@@ -28,6 +28,17 @@ <h1 class="i18n_name"></h1>
       </div>
     </header>
 
+    <!-- arrow showing users where to pin their extension, only shown on chromium -->
+    <div id="pin-nudge-arrow">
+      <div>
+        <a href="" id="dismiss-nudge" role="button" aria-label="i18n_report_close">
+          <img src="../icons/close.svg" alt="">
+        </a>
+        <p id="pin-nudge-text"><span id="pin-nudge-cta" class="i18n_intro_pin_action"></span> <span class="i18n_intro_pin_purpose"></span></p>
+      </div>
+      <img id="pin-image" src="images/pinning-instructions-4.png" alt="TO DO">
```

Comment:
The alt text should be blank if this is a decorative image, and should properly describe what it is we mean to convey otherwise. So here we should probably say something like:

To pin Privacy Badger's icon, click on the "puzzle piece" icon in your browser toolbar. Then, click on the "pin" next to Privacy Badger.

Reply:
Don't need the "-4" in the name?

Reply:
Oops yes thanks for the alt text, I forgot to come back to my TODO!